pkgs/applications/networking/twtxt/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "twtxt";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jointwt";
+    repo = pname;
+    rev = version;
+    sha256 = "15jhfnhpk34nmad04f7xz1w041dba8cn17hq46p9n5sarjgkjiiw";
+  };
+
+  vendorSha256 = "1lnf8wd2rv9d292rp8jndfdg0rjs6gfw0yg49l9spw4yzifnd7f7";
+
+  # Seems to have been put there during build.
+  postInstall = ''
+    rm $out/bin/tests
```

Comment:
Another way to do this (preferably) is to specify the subPackages to restrict what it's building

Reply:
```
subPackages = [ "binary" ];```